src/quo2/components/button.cljs

```diff
@@ -105,9 +107,10 @@
   [_ _]
   (let [pressed (reagent/atom false)]
     (fn [{:keys [on-press disabled type size before after above width
-                 on-long-press accessibility-label icon style]
+                 on-long-press accessibility-label icon style rounded]
```

Comment:
I changed the `plus-button` component to use quo2.button, it is fully rounded so I added this use case

Reply:
mhm I'm not sure it makes sense because quo2.button is a special component that can be used only with type and size params, its better to have plus-button as a separate component

Reply:
Cool, I removed it, just made it rounded by puting it inside a wrapping view